### ISSUE TITLE
Use of strikethrough

### ISSUE THREAD

**Issue by scottcwilson** [zencart/zencart#6802]
Sales and specials use strikethrough to show the original price of an item. 

The problem is, strikethrough is considered not accessible, and some scanners complain about this. 
(Reference: https://www.includia.com/blog/posts/strikethrough-accessibility/)

The easiest way to fix this seems to be wrapping the `normal` pricing tags in `includes/functions/functions_prices.php` (and `includes/classes/observers/ZcaBootstrapObserver.php` for Bootstrap) in `<del>`.  Although this doesn't satisfy Accessibe, it seems to be the solution screenreaders prefer.  

Thoughts? 

```
         if (!$pricing_handled) {
-            $show_normal_price = '<span class="normalprice">';
+            $show_normal_price = '<span class="normalprice"><del>';
             $show_normal_price .= $currencies->display_price($display_normal_price, zen_get_tax_rate($product_check->fields['products_tax_class_id']));
-            $show_normal_price .= ' </span>';
+            $show_normal_price .= ' </del></span>';
```

**Comment by drbyte** [comment]
I would expect screenreaders to be more consistent in their handling of something this significant.

Semantically a strikethru is quite clear in its intention. It doesn't have as many obscure interpretations as some other markup that was deemed semantically "wrong" enough to rewrite many HTML specs.

**Comment by drbyte** [comment]
Given that it hasn't been raised during the many years Zen Cart has been around, where does it land in terms of perceived value?

**Comment by drbyte** [comment]
I note that the examples in the linked site recommend pairing `<del>` with `<ins>`, not merely adding `<del>`.

**Comment by scottcwilson** [comment]
> Semantically a strikethru is quite clear in its intention. 

I agree `<s>` and `</s>` are semantically clear but we use CSS to do strikethrough. 

What makes this all especially miserable is that there's no agreed upon definition, and associated free checker, of how you get to AA.

**Comment by dbltoe** [comment]
Unfortunately, it's the screen readers that are the cause of the problem.  Some guidance;

Using strikethrough text in content is not inherently a violation of ADA (Americans with Disabilities Act) or WCAG (Web Content Accessibility Guidelines) standards, but there are some important considerations to keep in mind to ensure accessibility:

1. **Screen Readers**: Some screen readers may not convey the meaning of strikethrough text properly. Screen readers typically read strikethrough text without indicating that it is struck through, which can be confusing if the strikethrough conveys important information.

2. **Clarity**: Strikethrough is often used to show text that is no longer valid or relevant. If strikethrough is used to communicate critical information, it could be confusing for users with cognitive disabilities or those using assistive technologies. It's better to supplement strikethrough text with additional context, like an explanation or alternative visual cues.

3. **Color and Contrast**: If strikethrough is used in combination with low-contrast colors or other styling that reduces readability, this could present an issue under WCAG guidelines, specifically those related to text readability and color contrast.

### Recommendations:
- **Provide context**: If using strikethrough to indicate something like a correction or update, make sure that the surrounding text provides context.
- **Avoid reliance on styling alone**: Don’t rely solely on the strikethrough effect to convey important information. Ensure the meaning is clear through text or other accessible methods.
- **Test with screen readers**: If possible, test your site or app with various screen readers to ensure that strikethrough text does not cause confusion.

By following these guidelines, you can ensure that using strikethrough doesn’t cause accessibility issues for users who rely on assistive technology.

**Comment by gburton** [comment]
```
<span class="visually-hidden">original price</span><s>$99</s>
<span class="visually-hidden">sale price</span><span class="text-danger">$50</span>
```

Or

```
<span class="sr-only">original price</span><s>$99</s>
<span class="sr-only">sale price</span><span class="text-danger">$50</span>
```
